gossip/privdata/pvtdataprovider.go

```diff
@@ -219,7 +221,7 @@ func (pdp *PvtdataProvider) RetrievePvtdata(pvtdataToRetrieve []*ledger.TxPvtdat
 	fetchStats.fromLocalCache = totalEligibleMissingKeysToRetrieve - len(pvtdataRetrievalInfo.remainingEligibleMissingKeys)
 
 	if len(pvtdataRetrievalInfo.remainingEligibleMissingKeys) == 0 {
-		pdp.logger.Infof("Successfully fetched all %d eligible collection private write sets for block [%d] %s", totalEligibleMissingKeysToRetrieve, pdp.blockNum, fetchStats)
+		pdp.logger.Infof("Successfully fetched or resolved as to reconcile later all %d eligible collection private write sets for block [%d] %s", totalEligibleMissingKeysToRetrieve, pdp.blockNum, fetchStats)
```

Comment:
Since this is a common Info message we'll want to ensure readability. I struggled to read it as-is, here's my suggestion to make it more readable:  

```suggestion
		pdp.logger.Infof("Successfully fetched (or added to reconciler) all %d eligible collection private write sets for block [%d] %s", totalEligibleMissingKeysToRetrieve, pdp.blockNum, fetchStats)
```

Same for subsequent occurrences.